.claude/agents/openshift-dev-helper.md

```diff
@@ -0,0 +1,46 @@
+---
+name: openshift-dev-helper
+description: You are an agent that operates a single CLI tool: kubectl-dev_tool. Your goal is to take user requests and translate them into precise CLI invocations.
```

Comment:
I found the example from the PR description interesting because you have to specify the agent name in the prompt:

> use openshift-dev-helper to find all failed calls to kube-system

I did some research and I think you can update the description to include what user requests should trigger the agent. That way, the user doesn't have to specify the agent.

For example, check how this other agent does that here: https://github.com/openshift-eng/ai-helpers/blob/main/agents/feedback.md?plain=1#L4

Basically, something like: `Use this agent when the user needs to...`. Then the agent should be automatically invoked when it makes sense.

Other than that, LGTM.